modules/student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
```

Comment:
- [ ] Also as mentioned by the previous reviewer you should remove this line as the `classroom` room attribute is an internal property of the `Student` class and shouldn't be exposed to the outside world 😁 😁